src/CC2Edit/TileInspector.h

```diff
@@ -26,8 +26,9 @@ class QComboBox;
 class QSpinBox;
 class QCheckBox;
 class QGroupBox;
+class QAction;
+class QToolBar;
 class CC2ETileset;
-

```

Comment:
This blank line is intentional to separate the class forward declarations from the class definition below.